plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/http/TestHttpServer.java

```diff
@@ -154,6 +161,14 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
           requestHeaders.put(header, request.getHeader(header));
         }
 
+        StringBuilder body = new StringBuilder();
+        try (Reader reader = request.getReader()) { 
+          for (int c = reader.read(); c != -1; c = reader.read()) {
+            body.append((char) c);
+          }
+          requestBody = body.toString();
```

Comment:
I think you could simplify this with `CharStreams.toString(request.getReader())`